gui/templates/system/update_check.html

```diff
@@ -1,19 +1,27 @@
 {% extends "freeadmin/generic_form.html" %}
 {% block onSubmit %}
-doSubmit({
-    form: this,
-    event: e,
-    url: '{{ request.path }}',
-    longRunning: true,
-    progressbar: {
-      steps: [
-         {"label": gettext("Downloading packages")},
-         {"label": gettext("Installing packages")}
-      ],
-      poolUrl: '{% url "system_update_progress" %}',
-      fileUpload: false
+
+    doSubmit({
+        form: this,
+        event: e,
+        url: '{{ request.path }}',
+        longRunning: true,
+        progressbar: {
+          steps: [
+             {"label": gettext("Downloading packages")},
+             {"label": gettext("Installing packages")}
+          ],
+          poolUrl: '{% url "system_update_progress" %}',
+          fileUpload: false
+        }
+    });
```

Comment:
Lets revert these gratuitous changes of indentation please, it makes review harder. Remember this is a stable branch, the minimal changes the better.